### PR TITLE
Added Google Inbox cheatsheet

### PR DESCRIPTION
**Pull request by JamyGolden**
**What does your Instant Answer do?**
Adds content to the Cheatsheet instant answers.

**What is the data source for your Instant Answer? (Provide a link if possible)**
[https://support.google.com/inbox/answer/6080523](https://support.google.com/inbox/answer/6080523)
One can access a cheatsheet from within Google Inbox itself, that link explains how.

**Why did you choose this data source?**
It comes directly from Google.

**Are there any other alternative (better) data sources?**
No there are not.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Computer power users.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No.

**Are you having any problems? Do you need our help with anything?**
Not yet :)

**Where did you hear about DuckDuckHack? (For first time contributors)**
From the Duckduckgo home page.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
Screenshot

-----
IA Page: https://duck.co/ia/view/google_inbox_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
thanks for your contribution!

Can you please fill out the [pull request template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_cheat_sheet.md) and edit your initial comment. Once completed, we'll be sure to review your PR and give you some feedback.

Thanks!

File: share/goodie/cheat_sheets/inbox.json [hunk 1]
```diff
@@ -0,0 +1,138 @@
+{
+    "id": "inbox_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/inbox.json]: i think the ID for this one should be a little more specific: `google_inbox_cheat_sheet` seems better.

**Comment by JamyGolden** [comment]
@moollaza Thanks, yes you're right that does make more sense. I've changed filename, id and name of the cheatsheet and have updated the initial comment.

**Comment by moollaza** [comment]
>Sorry about the extra commits, I'm still getting used to working in Codio. Busy downloading the vagrant box, I think I may prefer that.

It's no problem at all. I don't really too much about the commits as long as they have useful messages and the final result is good :+1:

**Comment by moollaza** [comment]
@JamyGolden this LGTM :+1:

**Pull request merged**